README.md

```diff
@@ -13,6 +13,16 @@ configured by [eksctl](https://eksctl.io) through GitOps.
 - [Fluentd](https://www.fluentd.org/) & Amazon's [CloudWatch agent](https://aws.amazon.com/cloudwatch/) -- for cluster & containers' [log collection, aggregation & analytics in CloudWatch](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-setup-logs.html).
 - [podinfo](https://github.com/stefanprodan/podinfo) --  a toy demo application.
 
+## Pre-requisites
+
+A running EKS cluster with [IAM policies](https://eksctl.io/usage/iam-policies/) for:
+
+- ALB ingress
+- auto-scaler
+- CloudWatch
+
+[Here](https://github.com/weaveworks/eksctl/blob/master/examples/eks-quickstart-app-dev.yaml) is a sample `ClusterConfig` manifest for such policies enabled.
```

Comment:
_"... that shows how to enable all of these policies."_

We might want to add a note to that these will be scoped to a particular nodegroup, so a cluster with multiple nodegroups will require all of the add-on pods in question to have node selector at very least.

Reply:
> a cluster with multiple nodegroups will require all of the add-on pods in question to have node selector at very least.

@errordeveloper why is this?
What is the typical use-case for a cluster with multiple nodegroups?
Shouldn't we add such labels in our templates, add an optional flag for the target nodegroup's name, and deal with this when applying the profile in `eksctl`?
(If so, feel free to answer the above questions directly in a new issue. 🙇)

Reply:
Note that I've added some notes (PTAL 🙇) about the nodegroup scope and what to do with multiple nodegroups, but it is only based on my assumptions and my (limited) understanding. (Hence the aforementioned questions.)